lib/world_cup.rb

```diff
@@ -0,0 +1,38 @@
+class WorldCup
+  attr_reader :year,
+              :teams
+
+  def initialize(year, teams)
+    @year = year
+    @teams = teams
+  end
+
+  def active_players_by_position(position)
+    team_eliminated?
+    @teams.map do |team|
+      team.players_by_position(position)
+    end.flatten
+  end
+
+  def team_eliminated?
```

Comment:
This is deleting teams from your teams array, so it works, but we would rather keep that @teams array in tact and get the non eliminated teams using a find_all